### PR TITLE
PCHR-1556: Manage groups design issues

### PR DESCRIPTION
**Pull request by brunoqueiros**
This PR fix some design issues on manage groups page.

## Problems
- Style _ADD GROUP_ buttons
- Remove border and bakground from `.help`box
- Change action links color to `#42AFCB`
- Style table pagination
- Remove box shadow from new group modal container

***Before***
image

image

***After***
image

image

### CODE REVIEW

File: org.civicrm.bootstrapcivicrm/scss/civicrm/common/_buttons.scss [hunk 1]
```diff
@@ -39,7 +39,7 @@ $button-border-radius: 2px;
      * :not(.ui-dialog .button) is used to do not apply the selector .btn-secondary-outline
      * on modal's buttons.
      */
-    &:not(.crm-form-submit):not(.crm-close-accordion):not(.crm-clear-link):not(.ui-button) {
+    &:not(.crm-form-submit):not(.crm-close-accordion):not(.crm-clear-link):not(.ui-button):not(.button.newGroup) {
```
Comment (deb1990) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/common/_buttons.scss]: Is it .button or button?

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_manage-groups.scss [hunk 2]
```diff
@@ -143,6 +167,9 @@
   .CRM_Group_Form_Edit {
     .crm-group-form-block {
       padding: 20px;
+      box-shadow: none;
+      border: 1px solid #d3dee2;
```
Comment (deb1990) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_manage-groups.scss]: Use variables

File: org.civicrm.bootstrapcivicrm/scss/civicrm/common/_buttons.scss (hunk 1 above)
Comment (brunoqueiros) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/common/_buttons.scss]: .button

File: org.civicrm.bootstrapcivicrm/scss/bootstrap/overrides/_variables.scss [hunk 3]
```diff
@@ -55,6 +55,8 @@ $crm-brand-primary-lighter: lighten(desaturate(adjust-hue(#3ab1c9, -10.0699), 36
 
 $crm-dark-blue:         #586277 !default;
 
+$dialog-container-border: #d3dee2;
```
Comment (deb1990) [inline org.civicrm.bootstrapcivicrm/scss/bootstrap/overrides/_variables.scss]: Please use crm prefix for user defined variables

**Pull request merged**